src/engine/opengl.rs

```diff
@@ -11,7 +11,7 @@ use image::RgbaImage;
 
 use gl::types::*;
 
-/// The OpenGl context of our rendering pipeline. Contains the
+/// The `OpenGl` context of our rendering pipeline. Contains the
```

Comment:
The "OpenGl" here isn't a struct or anything, just the name of the technology. I don't have a strong preference here, but quoting it in backtics makes it read like it's a struct somewhere within the code (which it isn't). So I'd leave this alone.

Reply:
I will revert it.